assets/js/nav.js

```diff
@@ -13,22 +14,18 @@ window.addEventListener('resize', function() {
 });
 
 /**
- *  Hides the navigation menu items on mobile unless the user clicks on a link.
+ *  Displays the navigation menu items. 

```

Comment:
Can we change the method comment to include the fact that the method both displays and hides navigation menu items? Yes, nit-picky, I know.